CommonTools/UtilAlgos/interface/ObjectSelectorBase.h

```diff
@@ -54,7 +55,7 @@ class ObjectSelectorBase : public Base {
 private:
   /// process one event
   bool filter(edm::Event& evt, const edm::EventSetup& es) override {
-    Init::init(selector_, evt, es);
+    selectorInit_.init(selector_, evt, es);
     using namespace std;
```

Comment:
Please, remove this line

Reply:
Is it the `using namespace std;` line you want removed?

Reply:
Hi @Dr15Jones, yes, please. CMSbot pointed it as in conflict with the cms code rules